docker-compose.yaml

```
@@ -151,7 +152,7 @@ services:
     environment:
       - KUBECONFIG=/cluster.conf
       - DEBUG=True
-      - USERNAME=admin@test.com
+      - USERNAME=event_user
```

```suggestion
      - USERNAME=event_user@test.com
```
Shouldn't it be like this?

---

Yes, thanks!